src/sass/mixins.sass

```diff
@@ -1,3 +1,11 @@
+@import "variables"
+
 @mixin arwTransform($prop)
   transform: rotate($prop)
   -webkit-transform: rotate($prop)
+
+@mixin upper-navigation-tablet-up
```

Comment:
I really don't like this mixin, nor name, nor position, nor functionality

Reply:
[And what about this code?](https://github.com/JuliaKH/arsenal/pull/31/commits/5b524a59b75cff78d5ff70e23e5a1b4e63a9d2c8?file-filters%5B%5D=.css&file-filters%5B%5D=.map&file-filters%5B%5D=.sass#r300267334)
[](url)